spec/library_spec.rb

```diff
@@ -0,0 +1,177 @@
+require './lib/library.rb'
+require 'date'
+require 'yaml'
+
+describe Library do
+  let(:list) { subject.load_yaml('./lib/testYaml.yml') }
```

Comment:
Are you sure you have to do this. It looks like `load_yaml` is called in Library `initialize`?